classes/controller/menu/class-controller.php

```diff
@@ -49,5 +48,24 @@ public function load() {
 		 * @return void
 		 */
 		abstract public function create_admin_menu();
+
+		/**
+		 * Get underscore template from filesystem.
+		 *
+		 * @param string $template Template name.
+		 *
+		 * @return bool|string
+		 */
+		protected function get_template( $template ) {
```

Comment:
Do we need to add this function? It looks like it's not used anywhere.

Reply:
It's not in use. good catch 👍